docs/reference/elasticsearch/configuration-reference/thread-pool-settings.md

```diff
@@ -35,7 +35,12 @@ $$$search-throttled$$$`search_throttled`
 `write`
 :   For write operations and ingest processors. Thread pool type is `fixed` with a size of [`# of allocated processors`](#node.processors), queue_size of `max(10000, (`[`# of allocated processors`](#node.processors)`* 750))`. The maximum size for this pool is `1 + `[`# of allocated processors`](#node.processors).
 
-`write_coordination`
+  :::{note}
+  In {{stack}} 9.0 and earlier, the `write` thread pool was also used for bulk requests.
+  In {{stack}} 9.1 and earlier, the queue_size was 10000.
+  :::
```

Comment:
Strangely, your `:::` is rendering in the output.

<img width="694" height="208" alt="screen" src="https://github.com/user-attachments/assets/ce2fc3b2-e953-42f0-a0d6-087f5543c59d" />